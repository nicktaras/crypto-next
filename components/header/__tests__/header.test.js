/* eslint-env jest */
import { shallow } from 'enzyme'
import React from 'react'
import renderer from 'react-test-renderer'
import Header from '../header.jsx'

describe('With Enzyme', () => {
  it('should have the correct logo path', () => {
    const app = shallow(<Header />)
    expect(app.find('img').prop("src")).toEqual('./../static/logo.svg')
  })
})

describe('With Snapshot Testing', () => {
  it('should have the logo svg within the header', () => {
    const component = renderer.create(<Header />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
