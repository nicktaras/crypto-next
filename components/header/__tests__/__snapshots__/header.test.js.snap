// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With Snapshot Testing should have the logo svg within the header 1`] = `
<header
  className="jsx-4005495942"
>
  <div
    className="jsx-4005495942 wrapper"
  >
    <img
      className="jsx-4005495942"
      src="./../static/logo.svg"
    />
    <h1
      className="jsx-4005495942"
    >
      Crypto-Watch
    </h1>
  </div>
</header>
`;
